tools/monitoring/mysql/db_mysql.go

```diff
@@ -118,3 +118,45 @@ func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumb
 
 	return tx.Commit()
 }
+
+// CheckAlertCondition checks whether the given error pattern meets
+// the alert condition specified in config
+func CheckAlertCondition(errorPattern string, config *config.SelectedConfig, db *sql.DB) (bool, error) {
+	// the timestamp we want to start collecting logs
+	startTime := time.Now().Add(time.Minute * time.Duration(config.Period))
+
+	_, err := db.Query(
+		"CREATE VIEW Matched AS\nSELECT Jobname, PrNumber FROM ErrorLogs\nWHERE\n    ErrorPattern=? and\n    TimeStamp > ?",
```

Comment:
Remove `\n` in the query.

Reply:
I think \n is a better separator here than a space, though there is no semantic difference. Line separation is the norm in sql and makes the text more understandable

Reply:
I agree line separator (with actual new line, not escape character) is better than spaces. With the current style, I would consider this as a single line. Having escape character directly in the query is difficult to read as we have to do the escape char conversion in our head. Maybe make the actual query multi-line for better readability?

Reply:
Agree